### PR TITLE
fix abs namespace

### PR DESCRIPTION
**Pull request by untereiner**

### CODE REVIEW

File: src/coreComponents/finiteElement/elementFormulations/Qk_Hexahedron_Lagrange_GaussLobatto.hpp [hunk 1]
```diff
@@ -645,7 +645,7 @@ computeDampingTerm( int q,
   B[0] = J[0][0]*J[0][0]+J[1][0]*J[1][0]+J[2][0]*J[2][0];
   B[1] = J[0][1]*J[0][1]+J[1][1]*J[1][1]+J[2][1]*J[2][1];
   B[2] = J[0][0]*J[0][1]+J[1][0]*J[1][1]+J[2][0]*J[2][1];
-  return sqrt( abs( LvArray::tensorOps::symDeterminant< 2 >( B ) ) )*GL_BASIS::weight( qa )*GL_BASIS::weight( qb );
+  return sqrt( std::abs( LvArray::tensorOps::symDeterminant< 2 >( B ) ) )*GL_BASIS::weight( qa )*GL_BASIS::weight( qb );
```
Comment (TotoGaz) [inline src/coreComponents/finiteElement/elementFormulations/Qk_Hexahedron_Lagrange_GaussLobatto.hpp:648]: I believe you rather be using the `abs` function defined in `LvArray/src/math.hpp`?

Comment (untereiner) [reply src/coreComponents/finiteElement/elementFormulations/Qk_Hexahedron_Lagrange_GaussLobatto.hpp]: I guess I have to change it a few lines over it (line 628) too ?

Comment (TotoGaz) [reply src/coreComponents/finiteElement/elementFormulations/Qk_Hexahedron_Lagrange_GaussLobatto.hpp]: You most probably do.

**Review by TotoGaz — APPROVED** [review]

**Comment by francoishamon** [comment]
I am merging this 2-line PR which is at the top of the merge queue and fixes a compilation issue

**Comment by cj215** [comment]
I am still getting the same error after pulling this commit. The files concerned are:

GEOSX/src/coreComponents/physicsSolvers/wavePropagation/ElasticWaveEquationSEM.cpp
GEOSX/src/coreComponents/finiteElement/FiniteElementDispatch.hpp
GEOSX/src/coreComponents/physicsSolvers/wavePropagation/ElasticWaveEquationSEM.cpp
GEOSX/src/coreComponents/physicsSolvers/wavePropagation/ElasticWaveEquationSEMKernel.hpp

**Comment by francoishamon** [comment]
> I am still getting the same error after pulling this commit. The files concerned are:
> 
> GEOSX/src/coreComponents/physicsSolvers/wavePropagation/ElasticWaveEquationSEM.cpp GEOSX/src/coreComponents/finiteElement/FiniteElementDispatch.hpp GEOSX/src/coreComponents/physicsSolvers/wavePropagation/ElasticWaveEquationSEM.cpp GEOSX/src/coreComponents/physicsSolvers/wavePropagation/ElasticWaveEquationSEMKernel.hpp

Ok yes, I see what happened: [this PR](https://github.com/GEOSX/GEOSX/commit/794dbf0844aaff7e1448dedc226e5fbeba9a279c) has added more `abs` that should have been `LvArray::math::abs` in the files that you listed. We have to do another PR to change them as well.

**Pull request merged**